### ISSUE TITLE
Couple of issues on the first try

### ISSUE THREAD

**Issue by DooMl** [nirewen/discord-netflix#45]
#### Issue description
1. Media continues playing after leaving the player

2. Interactive episodes show playing time

#### Steps to reproduce the issue

1. Play any movie/show, leave the player and check

2. Play any interactive episode ( tested on Minecraft ) and check

#### What's the expected result?

1. RPC changes to "Browsing in the catalogs"

2. No playing time on Discord

#### What's the actual result?

1. Keeps showing the movie/episode

2. Keeps showing the movie/episode but without remaining time

#### Additional details / screenshot

1. None

2. ![Screenshot](http://prntscr.com/lpnwv2)

**Comment by NovusTheory** [comment]
Can confirm there is some issues with interactive videos. I might put up a PR for this issue shortly.

**Comment by DooMl** [comment]
I just noticed some interactive shows don't even play...

**Comment by NovusTheory** [comment]
Yes, the issue is that interactive video's may not have the data necessary to provide a playing status properly or that the interactive video is doing something different from native Netflix video usage that might be causing some problems.
